pkg/services/ngalert/api/api_ruler.go

```diff
@@ -486,8 +486,10 @@ func toGettableExtendedRuleNode(r ngmodels.AlertRule, namespaceID int64, provena
 		},
 	}
 	forDuration := model.Duration(r.For)
+	forErrorDuration := model.Duration(r.ForError)
 	gettableExtendedRuleNode.ApiRuleNode = &apimodels.ApiRuleNode{
 		For:         &forDuration,
+		ForError:    &forErrorDuration,
```

Comment:
I think `ForError` needs to be in `GrafanaManagedAlert` instead of `ApiRuleNode` as `ForError` is not Prometheus compatible.